clients.json

```diff
@@ -544,6 +544,16 @@
     "authors": ["vitaliykhamin"]
   },
 
+  {
+    "name": "laserdisc",
+    "language": "Scala",
+    "repository": "https://github.com/laserdisc-io/laserdisc",
+    "description": "Future free Fs2 native pure FP Redis client http://laserdisc.io",
+    "authors": ["sirocchj", "barambani"],
```

Comment:
This should be a Twitter handle (not GitHub) and 'sirocchj' does not exist.

Reply:
User `sirocchj` do exist https://github.com/sirocchj.

But 👍 for the pointer - added the twitter handle.